ingest/sqs/pools/transformer/pool_transformer.go

```diff
@@ -100,35 +93,29 @@ var uosmoRoutesFromDenom map[string][]poolmanagertypes.SwapAmountOutRoute = map[
 var stablesOverwrite map[string]struct{} = map[string]struct{}{
 	// Tether USD (Wormhole)
 	usdtDenom: {},
+	usdcDenom: {},
 }
 
 var _ domain.PoolsTransformer = &poolTransformer{}
 
 // NewPoolTransformer returns a new pool ingester.
-func NewPoolTransformer(assetListGetter domain.AssetListGetter, keepers domain.SQSIngestKeepers) domain.PoolsTransformer {
+func NewPoolTransformer(keepers domain.SQSIngestKeepers, defaultUSDCUOSMOPoolID uint64) domain.PoolsTransformer {
 	return &poolTransformer{
 		gammKeeper:         keepers.GammKeeper,
 		concentratedKeeper: keepers.ConcentratedKeeper,
 		cosmWasmKeeper:     keepers.CosmWasmPoolKeeper,
 		bankKeeper:         keepers.BankKeeper,
 		protorevKeeper:     keepers.ProtorevKeeper,
 		poolManagerKeeper:  keepers.PoolManagerKeeper,
-		assetListGetter:    assetListGetter,
+
+		defaultUSDCUOSMOPoolID: defaultUSDCUOSMOPoolID,
 	}
 }
 
 // processPoolState processes the pool state. an
 func (pi *poolTransformer) Transform(ctx sdk.Context, blockPools domain.BlockPools) ([]sqsdomain.PoolI, sqsdomain.TakerFeeMap, error) {
-	goCtx := sdk.WrapSDKContext(ctx)
-
-	// TODO: can be cached
-	tokenPrecisionMap, err := pi.assetListGetter.GetDenomPrecisions(goCtx)
-	if err != nil {
-		return nil, nil, err
-	}
```

Comment:
Note: The asset list is not needed anymore and can be removed. Now, we only depend on it in `sqs`